### PR TITLE
Fix sample status count for projects

### PR DESCRIPTION
**Pull request by jenniferboedker**
Fixes the number of samples in a status and the shown samples in the sample view to only consider NGS samples for which we can provide reliable tracking information

### CODE REVIEW

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectView.java [hunk 1]
```diff
@@ -137,6 +137,7 @@ private void selectProject(Project project) {
       removeFileDownloaders(downloadButton);
     }
     if (project.projectStatus().totalCount() < 1) {
+      samplesButton.setEnabled(false);
```
Comment (wow-such-code) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectView.java:140]: why is this necessary? are such projects shown at all?

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/data/OpenBisConnector.java [hunk 2]
```diff
@@ -43,64 +44,73 @@ public OpenBisConnector(Credentials credentials, PortalUser portalUser,
...
+    return cachedProjectSamples.stream().map(convertToSample()).collect(toList());
+  }
+
+  private void updateCache() {
+
+    SampleSearchCriteria isNgs = new SampleSearchCriteria();
+    isNgs.withOrOperator();
+    isNgs.withProperty("Q_SAMPLE_TYPE").thatContains("DNA");
+    isNgs.withProperty("Q_SAMPLE_TYPE").thatContains("RNA");
+    // this guarantees that no q-entity is contained and only test samples have Q_SAMPLE_TYPE
```
Comment (wow-such-code) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/data/OpenBisConnector.java:78]:
```suggestion
    // this guarantees that no q-entity is contained as only test samples have Q_SAMPLE_TYPE
```

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/data/OpenBisConnector.java [hunk 3]
```diff
@@ -43,64 +44,73 @@ public OpenBisConnector(Credentials credentials, PortalUser portalUser,
...
+
+    SampleFetchOptions fetchOptions = new SampleFetchOptions();
+    fetchOptions.withType();
+    fetchOptions.withProperties();
+    fetchOptions.withProject();
```
Comment (wow-such-code) [inline sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/data/OpenBisConnector.java:83]:
```suggestion
```
can be removed as it does not work/is not used

**Review by wow-such-code — DISMISSED** [review]

File: sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectView.java (hunk 1 above)
Comment (jenniferboedker) [reply sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/view/projects/ProjectView.java:140]: yes they are, otherwise you can navigate to an empty samples view.

**Comment by jenniferboedker** [comment]
@wow-such-code I accepted your suggestions, can you approve again? :)

**Review by KochTobi — APPROVED** [review]
LGTM
